### DISCUSSION TITLE
Animation not works on android

### DISCUSSION THREAD

**Discussion by razamsalem** [software-mansion/react-native-reanimated#5837] [Q&A]
Hey,
 I'm using react-native-reanimated for animated comments that enter the chat, The problem is that the animation works for iOS but for android there is no animation at all. 
 
 This is the code I'm using:
 
```tsx
  <Animated.View
          entering={SlideInRight.duration(200)}
          exiting={SlideOutRight.duration(200)}
        >
          <TouchableOpacity onPress={() => setIsCommentInput(true)}>
            {!isCommentInput && !isLandscape && (
              <CommentIcon
                style={{
                  position: 'absolute',
                  bottom: SIZES.xSmall,
                  right: 22,
                  zIndex: 99,
                }}
              />
            )}
          </TouchableOpacity>
        </Animated.View>
```
 
 Why can it work only for iOS?

**Comment by tomekzaw** [comment]
@razamsalem Do you have reduced motion enabled in the system settings on Android?

**Reply by razamsalem** [reply]
Thanks for the response
No, the animation in the settings is on

**Comment by MatiPl01** [comment]
Hey, are you sure that the animation works for iOS and doesn't work only on Android? I think you should specify `entering` and `exiting` animations on your removed component (i.e. `CommentIcon`), not on its parent, which is not being removed/added.

But, the behavior should be the same on iOS and Android. I tested code similar to yours and it doesn't work on iOS as well.
